modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/schema/IgniteSchema.java

```diff
@@ -88,4 +91,33 @@ public void removeTable(String tblName) {
     public void addFunction(String name, Function func) {
         funcMap.put(name, func);
     }
+
+    /**
+     * @param name View name.
+     * @param sql View sql.
+     */
+    public void addView(String name, String sql) {
+        viewMap.put(name, sql);
+    }
+
+    /**
+     * @param name View name.
+     */
+    public void removeView(String name) {
+        viewMap.remove(name);
+    }
+
+    /**
+     * Registers current {@code IgniteSchema} in parent {@code schema}.
```

Comment:
Let's emphasize that parent schema is a SchemaPlus: `{@code schema}` -> `{@code SchemaPlus}`